src/doc/trpl/ownership.md

```diff
@@ -174,7 +173,7 @@ fn foo(v: Vec<i32>) -> Vec<i32> {
 }
 ```
 
-This would get very tedius. It gets worse the more things we want to take ownership of:
+This would get very tedius. It gets the worse the more things we want to take ownership of:
```

Comment:
O, I am not a native English speaker. I thought this was a case like in this phrase from the British Corpus: "the bigger the misperception, the worse the outcome around". Now I see that the phrase "gets worse the..." is something different. I am reverting this.